glib/Cargo.toml

```diff
@@ -32,6 +32,7 @@ gobject_ffi = { package = "gobject-sys", path = "gobject-sys" }
 glib-macros = { path = "../glib-macros" }
 rs-log = { package = "log", version = "0.4", optional = true }
 smallvec = "1.0"
+thiserror = "1"
```

Comment:
I strongly vote against this. The gtk-rs crates are already big enough to not add more dependencies.

Reply:
Other crates of gtk-rs seems to use it already iirc

Reply:
`cairo` and `gio` and `gstreamer`

Reply:
Mmmm, that's already used by gio and cairo, so it's not really a new dependency

Reply:
I'll remove it from these crates. Please remove it.

Reply:
Good luck getting the `Error` and `Display` impls right then :) I wonder how often we'll have this very discussion

Reply:
We have very different positions on this unfortunately. I don't think we'll ever agree. As long as I provide the PRs, I guess it's fine? ;)

Did this one already: https://github.com/gtk-rs/gtk-rs-core/pull/415

Reply:
You can keep it here FWIW